docs/integrations/microsoft-azure/sql.md

```diff
@@ -85,85 +83,69 @@ _sourceCategory=Azure/DB/SQL/Logs ErrorEvent "\"operationName\":\"ErrorEvent\""
+	8. **Event hub name (optional).** If you have chosen Method 1 (Azure Event Hub Source) for collecting logs, select the event hub name, which you created manually, or if you have chosen Method 2 (Collect logs from Azure monitor using Azure functions), then select **insights-operational-logs**.
+	9. Select **RootManageSharedAccessKey** from **Select event hub policy name** dropdown.
+	10. Select the **Audit** label under **Logs**.
+	11. Click **Save**.
+
+### Collect metrics from Azure Monitor by streaming to EventHub
 
 In this step, you configure a pipeline for shipping metrics from Azure Monitor to an Event Hub, on to an Azure Function, and finally to an HTTP Source on a hosted collector in Sumo Logic. The pipeline is described on [Collect Metrics from Azure Monitor](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor.md).
 1. Perform Steps 1 and Step 2 of [Collect Metrics from Azure Monitor.](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor.md)   
-In Step 1, you create an HTTP source. When you configure the, plan your source category to ease the querying process.  A hierarchical approach allows you to make use of wildcards. For example: \
+In Step 1, you create an HTTP source. When you configure the, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: \
 `Azure/DB/SQL/Metrics`
 2. Push metrics from Azure Monitor to Event Hub.
-   * From the left pane, select **ALL Services.
-   * Search for and select "Monitor".
+   * From the left pane, select **ALL Services**.
+   * Search for and select **Monitor**.
    * In the **Monitor** pane, select **Diagnostic Settings **under **Settings**.
-   * Select the resource for which you want to export metrics. If **diagnostics** is not enabled click **Turn on Diagnostics Settings.
+   * Select the resource for which you want to export metrics. If **diagnostics** is not enabled click **Turn on Diagnostics Settings**.
    * Once diagnostics are enabled, click **Add a diagnostic setting**.
    * The **Diagnostic Settings** page appears.
    * In the left pane:
       * Enter a name for the diagnostic setting.
-      * Click the  **Stream to an event hub **checkbox.
+      * Click the **Stream to an event hub** checkbox.
       * Select **Configure event hub.** The right pane appears.
    * In the right pane:
       * Choose a **Subscription**.
       * Select `SumoMetricsNamespace_<UniqueSuffix>` as the event hub namespace.
       * Select **insights-metrics-pt1m** as the event hub name.
-      * Select an event hub policy name. You can use the default policy** RootManageSharedAccessKey**.
-   * Check the AllMetrics box.
+      * Select an event hub policy name. You can use the default policy **RootManageSharedAccessKey**.
+   * Check the **AllMetrics** box.
    * Click **OK**.
    * Save the **Diagnostics Setting**.
```

Comment:
In this entire section, you need to change from bulleted lists to numbered lists because it is a procedure.

Reply:
Thanks for catching this John, I have fixed this now.